ui: database details page

vladlos opened the pull request:
updated design on database details page and added statements table info

Resolves: #44303, #44764, #44765 

Release note(ui): none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/44601)
<!-- Reviewable:end -->

vladlos committed:
ui: database details page

updated design on database details page and added statements table info

Resolves: #44303

Release note(ui): none

dhartunian commented:
bors r+
